### DISCUSSION TITLE
Is this still maintained?

### DISCUSSION THREAD

**Discussion by SyncroIT** [remotely-save/remotely-save#264] [General]
It seems like the repo is dead. Is it worth contributing to it? Is there any maintainer merging new PRs?

**Comment by berot3** [comment]
Fair question, looking at the recent issues... @fyears seems to be off the project?

**Comment by hellodword** [comment]
Same question.

**Comment by uroybd** [comment]
Same question.

**Comment by tenebrisnox** [comment]
I hope it is still in development. I've found it super-useful and another step forward towards self-hosting all my data and not relying on Apple and Microsoft.

**Comment by StrangePeanut** [comment]
To add onto what's been said, I've found Remotely Save to be the ONLY community plugin solution to be reliable. With another community plugin, I'd regularly lose notes. I love how Remotely Save just works with no complicated configuration out of the box. It would be an absolute downer if the plugin loses support. It would make me stick with an older Obsidian build for as long as I can.

**Reply by uroybd** [reply]
> To add onto what's been said, I've found Remotely Save to be the ONLY community plugin solution to be reliable. With another community plugin, I'd regularly lose notes. I love how Remotely Save just works with no complicated configuration out of the box. It would be an absolute downer if the plugin loses support. It would make me stick with an older Obsidian build for as long as I can.

I would also like to add to this. This plugin is as fast as the official sync in my case and with added bonus of the data staying in the storage of my choice. I would like to contribute to this project both with code and to my ability with donations.

**Comment by xplosionmind** [comment]
Hello to you all. I have just started using this plugin, and I am very sad that its last update dates to **one year ago**…

Does anyone have any news?

**Comment by StrangePeanut** [comment]
fyears commented last week:

> Hi all. I am the plugin user.
> 
> Firstly thank you all for your support!!!
> 
> Admittedly I was too busy (well I DO have a job so that I could earn money and food...) and too lazy to update the plugin in the past few years..
> 
> Furthermore, I personally kept using my plugin to sync my notes over s3, so far so good. I did not directly encounter a reproducible bug by myself. Thus I do not have a very good reason to update the functions of the plugin.
> 
> But anyways, since I am likely to keep using the plugin unless anyone in community provides a better one. I will keep maintaining this plugin in the future!
> 
> BTW, livesync and obsidian-git doesn't meet my needs because the first one needs a db server and the second one do not support mobile. So I think they and remotely-save share the different "market" in community.

Source: https://github.com/remotely-save/remotely-save/issues/280#issuecomment-1528642132

**Reply by berot3** [reply]
Thanks! Didn't see it there 👍🏻

**Reply by berot3** [reply]
@SyncroIT might mark this one as answered/solved

**Comment by fyears** [comment]
yes everyone i am back to the party
